mmap_test.go

```diff
@@ -159,7 +174,130 @@ func TestReadWrite(t *testing.T) {
 	}()
 }
 
+func TestReadWriteString(t *testing.T) {
```

Comment:
Can we break this test into Read and Write?

Reply:
How'd that be useful? Currently we can write, read and check!

Reply:
Right now test case looks quite big. Breaking it into Read and Write test cases will improve readability.